features/networking/operator.feature

```diff
@@ -52,3 +52,134 @@ Feature: Operator related networking scenarios
     And evaluation of `cluster_operator('network').condition(type: 'Failing',cached: false)` is stored in the :failing_status_post_patch clipboard
     Then the expression should be true> cb.failing_status_post_patch["status"]=="True"
     And the expression should be true> cluster_operator('network').version_exists?(version: cb.ocp_version)
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-22201
+  @admin
+  Scenario: Should have a clusteroperator object created under config.openshift.io api group for network-operator
+    Given the master version >= "4.0"
+    # Check the operator object has version
+    Given the expression should be true> cluster_operator('network').versions.length > 0
+    # Check the operator object has status for Failing|Progressing|Available
+    And the expression should be true> cluster_operator('network').condition(type: 'Available')['status'] == "True"
+    And the expression should be true> cluster_operator('network').condition(type: 'Failing')['status'] == "False"
+    And the expression should be true> cluster_operator('network').condition(type: 'Progressing')['status'] == "False"
```

Comment:
You could use this expression inside the new step instead of the get commands. And then use the step here. But don't want to delay this pull request. You can make consistent later.